src/en/authors-charm-benchmarks.md

```diff
@@ -1,4 +1,4 @@
-Title: Benchmarking Juju services  
+Title: Benchmarking Juju applications  
```

Comment:
Yes, and I can see about helping with the larger one. As long as they're published searchable docs, however, I feel like updating them in line is reasonable.